website-custom-views/src/content/concepts/oauth-scopes-and-user-permissions.mdx

```diff
@@ -13,7 +13,7 @@ To fulfill these requirements, your Custom View would need the following OAuth S
 
 These [OAuth Scopes](https://docs-beta-custom-views.commercetools.vercel.app/api/scopes) must be specified in your Custom View config, using the [`oAuthScopes` field](/api-reference/custom-view-config#oauthscopes).
 
-```json title="custom-view-config.mjs"
+```js title="custom-view-config.mjs"
```

Comment:
Nit: hmm if we set the syntax to JS, we should probably adjust the content to be JS syntax and not JSON.

Reply:
Hi @emmenko, thanks. I’ve applied this change too all .mjs files in 4c0aefbc6c898033694c442f3d7af3bfe0944430